modules/auxiliary/scanner/http/trace.rb

```diff
@@ -27,36 +31,36 @@ def run_host(target_host)
     begin
       res = send_request_raw({
         'version'      => '1.0',
-        'uri'          => '/',
+        'uri'          => '/<script>alert(1337)</script>', #XST Payload
         'method'       => 'TRACE',
         'headers' =>
         {
-          'Cookie' => "did you echo me back?",
+          'Cookie' => "TRACE ME PLS",
         },
-      }, 10)
+      })
 
       if res.nil?
-        print_error("no repsonse for #{target_host}")
-      elsif (res.code == 200)
-        print_good("#{target_host}:#{rport}-->#{res.code}")
+        vprint_error("no repsonse for #{target_host}")
+      elsif res.code == 200 and res.body =~/>alert\(1337\)/
+        vprint_status("#{target_host}:#{rport}-->#{res.code}")
         print_good("Response Headers:\n #{res.headers}")
         print_good("Response Body:\n #{res.body}")
-        print_good("TRACE appears to be enabled on #{target_host}:#{rport} \n")
-        report_note(
+        print_good("#{target_host}:#{rport} is vulnerable to Cross-Site Tracing")
+        report_vuln(
           :host   => target_host,
           :port   => rport,
           :proto => 'tcp',
           :sname  => (ssl ? 'https' : 'http'),
           :type   => 'service.http.method.trace',
-          :data   => "TRACE method is enabled for this service",
```

Comment:
@void-in  told me to use :info instead of :data because of using report_vuln and not report_note